src/containers/tutor-profile/profile-info/ProfileInfo.styles.js

```diff
@@ -79,5 +79,29 @@ export const styles = {
     description: {
       typography: 'overline'
     }
+  },
+  wrapperForPhoto: {
+    display: 'flex',
+    gap: '10px'
+  },
+  avatarContainer: {
+    flex: 1,
+    maxWidth: '650px'
+  },
+  avatarContainerMobile: { flex: 1 },
+  linkToReviews: { color: 'text.primary' },
+  stylesForAppRating: {
+    variantOption: 'overline',
+    fontSize: '10px',
```

Comment:
maybe it would be better to use typography style property instead of fontSize, how do you think?

Reply:
I deleted this style completely. Fixed